src/admin/components/Fields.vue

```diff
@@ -648,45 +689,300 @@
                         </label>
                     </template>
                 </div>
-            </td>
+            </div>
         </template>
 
         <template v-if="'gmap' == fieldData.type && ! hideMap">
-            <th scope="row">
-                <label :for="sectionId + '[' + fieldData.name + ']'">{{ fieldData.label }}</label>
-            </th>
-
-            <td class="tooltips-data"></td>
-
-            <td v-bind:class="[fieldData.content_class ? fieldData.content_class : '']">
-                <input type="hidden" :name="sectionId + '[' + fieldData.name + ']'" :value="mapLocation">
-                <Mapbox
-                    v-if="mapApiSource === 'mapbox'"
-                    @hideMap="onHideMap"
-                    @updateMap="onUpdateMap"
-                    :accessToken="mapboxAccessToken"
-                    :location="mapLocation"
-                    width="100%"
-                    height="300px"
-                />
-                <GoogleMaps
-                    v-else
-                    @hideMap="onHideMap"
-                    @updateMap="onUpdateMap"
-                    :apiKey="googleMapApiKey"
-                    :location="mapLocation"
-                />
+            <div class="field_contents" v-bind:class="[fieldData.content_class ? fieldData.content_class : '']">
+                <fieldset>
+                    <div class="field_data">
+                        <h3 class="field_heading" scope="row">
+                            {{ fieldData.label }}
+                        </h3>
+                        <p class="field_desc" v-html="fieldData.desc"></p>
+                    </div>
+                </fieldset>
+                <div class="field gmap-field">
+                    <input type="hidden" :name="sectionId + '[' + fieldData.name + ']'" :value="mapLocation">
+                    <Mapbox
+                        v-if="mapApiSource === 'mapbox'"
+                        @hideMap="onHideMap"
+                        @updateMap="onUpdateMap"
+                        :accessToken="mapboxAccessToken"
+                        :location="mapLocation"
+                        width="100%"
+                        height="300px"
+                    />
+                    <GoogleMaps
+                        v-else
+                        @hideMap="onHideMap"
+                        @updateMap="onUpdateMap"
+                        :apiKey="googleMapApiKey"
+                        :location="mapLocation"
+                    />
+                </div>
                 <p v-if="hasError( fieldData.name )" class="dokan-error">
                     {{ getError( fieldData.label ) }}
                 </p>
-                <p class="description" v-html="fieldData.desc"></p>
-            </td>
+            </div>
         </template>
-    </tr>
+
+        <template v-if="'social' === fieldData.type">
```

Comment:
This component is too big. Can it be sliced into multiple components?
Also, here some similar things are being repeated by checking with `v-if`. See if a common component can be built to be used by passing the parameter `fieldData.type`.